modules/nf-core/delly/call/main.nf

```diff
@@ -1,35 +1,37 @@
 process DELLY_CALL {
     tag "$meta.id"
-    label 'process_medium'
+    label 'process_single'
```

Comment:
What is the reason behind this change?

Reply:
Delly doesn't use multiple CPUs, so the only reason to use `process_medium` would be memory usage. Memory requirements should be rather small for sample-wise calling and if `process_single` is insufficient, users can easily increase just the memory of the process without wasting CPUs.